### ISSUE TITLE
Fleck sign of elbow

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10806]
**Preferred term label:**
Fleck sign of elbow

**Synonyms**
Lateral elbow x-ray showing proximal avulsion fracture off of the olecranon; "flake sign" of the elbow; Dunn-Kusnezov Sign

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Triceps tendon rupture

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Just to check -- this is Flake sign (not "Fleck" as in the title?)

**Comment by MickeySegal** [comment]
It seems to be Fleck, though the orthopedics curator having added Flake in the synonyms suggests that both forms are used.

https://pmc.ncbi.nlm.nih.gov/articles/PMC5684929/ "The primary data points obtained were the rates of (1) presence of fleck fracture, [Dunn-Kusnezov Sign], on the lateral x-ray film...."

https://pmc.ncbi.nlm.nih.gov/articles/PMC9563226/ "In case of an avulsion fracture at the fibular insertion of the retinaculum, conventional radiographs show a flake of bone at the posterolateral distal fibula, designated as the ‘ankle fleck sign’.
"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001362
- label: Fleck sign of elbow
- parents: [Abnormal olecranon morphology (HP:0004032)](https://hpo.jax.org/app/browse/term/HP:0004032)
- pmid: PMID:28461956
- def: A radiograpzhic sign in which small bone fragments detached from the olecranon are seen on lateral radiographs in cases with avulsion of the tendon.
- comment: See Figure 2 of PMID:28461956.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10806
- synonyms: Dunn-Kusnezov sign(NONE)|Flake sign of the elbow(NONE)
